docs/building-a-form/creating-a-form-config-file.md

```diff
@@ -1,286 +1,173 @@
 # Creating a form config file
 
-Your form is generated from a JSON Schema configuration file called `form.js`, along with a few other key configuration files. For information about the basic files you need to configure a form, see "Create required files" in the *US Forms System Getting Started Guide*.
+Your form is generated from a JSON Schema configuration file called `form.js`, along with a few other key configuration files.
 
-### In this guide
-
-- [Example `form.js` file](#example-form.js-file)
-- [About the `schema` and `uiSchema` objects](#about-the-schema-and-uischema-objects)
-- [Configuring `uiSchema` using rjsf options](#configuring-uischema-using-rjsf-options)
-- [Configuring `uiSchema` using US Form System options](#configuring-uischema-using-us-form-system-options)
-
-### Example `form.js` file
-
-This example `form.js` file will get you started with building your form.
-
-```js
-{
-  // Prefix string to add to the path for each page.
-  urlPrefix: '',
-
-  // The introduction page component. To exclude an introduction page, remove this component.
-  introduction: IntroductionComponent,
-
-  // The confirmation page component that will render after the user successfully submits the form.
-  confirmation: ConfirmationComponent,
+- For information about the basic files you need to configure a form, see "[Create required files](../getting-started/installing-the-us-forms-system-in-an-existing-application.md#create-required-files)."
+- For an example `form.js` file, see "[Quick Start: Example `form.js` file](quick-start-example-formjs-file.md)"
 
-  // The prefix for Google Analytics events that are sent for different form actions.
-  trackingPrefix: '',
-
-  // The title of the form, rendered on all pages.
-  title: '',
-
-  // The subtitle of the form, usually the form number. The subtitle is rendered on all pages when there's also a title.
-  subTitle: '',
+### In this guide
```

Comment:
I still feel kind of conflicted about this file and the content it contains. The way I see it there are 2 categories of content to include in our docs:
1. Information someone needs to know in order to use the library and build a form
2. Information about how the library works under the hood, which may help someone create customizations on top of the default functionality that's included

I feel like a lot of the content in this file is only relevant for purpose 2, and could be overwhelming for a user trying to understand the basics still. I think keeping this file as it is in this PR is fine, because it's too complicated to start moving around this content too, but I'd like to revisit this in a separate PR.

Reply:
To be clear, I think the title of this file is way off at this point given the information it has now. Before @dmethvin-gov added what's now in the Quick Start topic, the content that was here was squarely in Category 1. It's all moved around now, and like you said, it's Category 2 at this point. I'll take a crack at it based on your confirmation.

Reply:
@annekainicUSDS: Given that this information is a bit "extra" (and about customization?), should it be in the same topic as what's in "[Creating custom fields and widgets](https://github.com/usds/us-forms-system/blob/schema-uischema/docs/building-a-form/creating-custom-fields-and-widgets.md#creating-custom-fields-and-widgets)"? 

It's hard for me to tell on my own which of this information is necessary to know in order to build a form using the library, and which is for extending the native features the library doesn't provide out-of-the-box.

Reply:
Hm I'm not sure it really belongs there either, I think it might belong in a separate file. I can take on refactoring this file once this PR is merged!